hadoop-hdds/interface-client/src/main/proto/DatanodeClientProtocol.proto

```diff
@@ -393,6 +394,7 @@ message  WriteChunkRequestProto  {
   required DatanodeBlockID blockID = 1;
   required ChunkInfo chunkData = 2;
   optional bytes data = 3;
+  optional int32 containerinstanceIndex = 4;
```

Comment:
containerinstanceIndex -> containerInstanceIndex

Reply:
Fixed with moving this field to the `DatanodeBlockID`